akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/scaladsl/GrpcReadJournal.scala

```diff
@@ -95,22 +119,29 @@ final class GrpcReadJournal private (
     with LoadEventQuery {
   import GrpcReadJournal.log
 
-  private def this(system: ExtendedActorSystem, settings: GrpcQuerySettings) =
+  private def this(
+      system: ExtendedActorSystem,
+      settings: GrpcQuerySettings,
+      protoAnyPrefer: ProtoAnySerialization.Prefer) =
     this(
       system,
+      // FIXME can/should we load descriptors from config?
```

Comment:
seems overkill, but it also means that the "ordinary" way of creating it via the GrpcReadJournalProvider will not be useful